src/lib.rs

```diff
@@ -194,7 +194,7 @@ mod lock {
 
 #[cfg(all(
     windows,
-    any(target_env = "msvc", all(target_env = "gnu", target_arch = "x86")),
+    any(target_env = "msvc", all(target_env = "gnu", any(target_arch = "x86", target_arch = "arm"))),
```

Comment:
If needed, I can omit this part from the PR since `armv7-pc-windows-gnullvm` is not (yet) an officially supported Rust target.

For reference, this was necessary to fix this build error:
```
error[E0433]: failed to resolve: could not find `dbghelp` in `super`
   --> /data/mxe/usr/x86_64-pc-linux-gnu/lib/rustlib/src/rust/library/std/src/../../backtrace/src/backtrace/dbghelp.rs:149:39
    |
149 |     let dbghelp = match super::super::dbghelp::init() {
    |                                       ^^^^^^^ could not find `dbghelp` in `super`
```

Reply:
Seems fine. I'm not seeing it cause any issues yet. I'll keep it in mind though if any issue turns up after this is merged.